app/metrics/CloudWatch.scala

```diff
@@ -40,44 +49,54 @@ object CloudWatch {
       .withValue(allStacks)
   )
 
-  def putOldCountRequest(count: Int): PutMetricDataRequest = {
+  private[metrics] def putRequest(metricName: String, value: Int): PutMetricDataRequest = {
     new PutMetricDataRequest()
       .withNamespace(namespace)
       .withMetricData {
         new MetricDatum()
           .withMetricName(metricName)
-          .withValue(count.toDouble)
+          .withValue(value.toDouble)
           .withDimensions(dimensions.asJava)
       }
   }
 
-  def getOldCountRequest: GetMetricStatisticsRequest = {
+  private def getRequest(metricName: String): GetMetricStatisticsRequest = {
     val now = DateTime.now(DateTimeZone.UTC)
     new GetMetricStatisticsRequest()
       .withNamespace(namespace)
       .withMetricName(metricName)
       .withDimensions(dimensions.asJava)
       .withPeriod(60 * 60 * 24)  // 1 day (24 hrs)
-      .withStartTime(now.minusDays(30).toDate)
+      .withStartTime(now.minusDays(60).toDate)
       .withEndTime(now.toDate)
       .withStatistics(Statistic.Maximum)
   }
 
-  def extractCountRequestData(result: GetMetricStatisticsResult): List[(DateTime, Double)] = {
+  private[metrics] def extractDataFromResult(result: GetMetricStatisticsResult): List[(DateTime, Double)] = {
     result.getDatapoints.asScala.toList.map { dp =>
       new DateTime(dp.getTimestamp) -> dp.getMaximum.toDouble
     }.sortBy(_._1.getMillis)
   }
 
-  def getOldCountData(client: AmazonCloudWatchAsyncClient)(implicit executionContext: ExecutionContext): Future[List[(DateTime, Double)]] = {
-    awsToScala(client.getMetricStatisticsAsync)(getOldCountRequest).map(extractCountRequestData)
+  private def getWithRequest(request: GetMetricStatisticsRequest)(implicit executionContext: ExecutionContext): Future[List[(DateTime, Double)]] = {
+    awsToScala(client.getMetricStatisticsAsync)(request).map(extractDataFromResult)
+  }
+
+  private def putWithRequest(request: PutMetricDataRequest) = {
+    awsToScala(client.putMetricDataAsync)(request)
   }
 
-  def attemptOldCountData(client: AmazonCloudWatchAsyncClient)(implicit executionContext: ExecutionContext): Attempt[Option[List[(DateTime, Double)]]] = {
-    val tmp: Future[Either[AMIableErrors, Option[List[(DateTime, Double)]]]] = getOldCountData(client).map(ds => Right(Some(ds)))
+  private def attemptWithRequest(request: GetMetricStatisticsRequest)(implicit executionContext: ExecutionContext): Attempt[Option[List[(DateTime, Double)]]] = {
+    val tmp: Future[Either[AMIableErrors, Option[List[(DateTime, Double)]]]] = getWithRequest(request).map(ds => Right(Some(ds)))
     Attempt.fromFuture(tmp){ case e =>
-      Logger.warn("Failed to fetch old instance count cloudwatch data", e)
+      Logger.warn("Failed to fetch CloudWatch data", e)
       Right(None)
     }
   }
+
+  def get(metricName: String)(implicit executionContext: ExecutionContext): Attempt[Option[List[(DateTime, Double)]]] = {
```

Comment:
I think I'd consider inlining the `attemptWithRequest` function here, since it doesn't add any purity benefit having them be separate and it's only used once.